js/controller.js

```diff
@@ -1362,9 +1362,22 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     showVolumeSliderBar: function() {
       this.state.volumeState.volumeSliderVisible = true;
+      if (Utils.isAndroid()) {
+        this.startHideVolumeSliderTimer();
+      }
       this.renderSkin();
     },
 
+    startHideVolumeSliderTimer: function() {
+        this.cancelTimer();
+        var timer = setTimeout(function() {
+          if(this.state.volumeState.volumeSliderVisible === true){
+            this.hideVolumeSliderBar();
+          }
+        }.bind(this), 3000);
+        this.state.timer = timer;
```

Comment:
can you check if a timer already exists, if it does then cancel it.  Then do the assignment.